src/openrct2/sprites.h

```diff
@@ -787,24 +787,23 @@ enum
     SPR_G2_SORT = SPR_G2_BEGIN + 76,
     SPR_G2_COPY = SPR_G2_BEGIN + 77,
     SPR_G2_PASTE = SPR_G2_BEGIN + 78,
-    SPR_G2_TAB_DISABLED = SPR_G2_BEGIN + 79,
 
-    SPR_G2_GIGA_RC_BOOSTER_NE_SW = SPR_G2_BEGIN + 80,
-    SPR_G2_GIGA_RC_BOOSTER_NW_SE = SPR_G2_BEGIN + 81,
-    SPR_G2_BM_BOOSTER_NE_SW = SPR_G2_BEGIN + 82,
-    SPR_G2_BM_BOOSTER_NW_SE = SPR_G2_BEGIN + 83,
+    SPR_G2_GIGA_RC_BOOSTER_NE_SW = SPR_G2_BEGIN + 79,
+    SPR_G2_GIGA_RC_BOOSTER_NW_SE = SPR_G2_BEGIN + 80,
+    SPR_G2_BM_BOOSTER_NE_SW = SPR_G2_BEGIN + 81,
+    SPR_G2_BM_BOOSTER_NW_SE = SPR_G2_BEGIN + 82,
 
-    SPR_G2_TOOLBAR_MUTE = SPR_G2_BEGIN + 86,
-    SPR_G2_TOOLBAR_MUTE_PRESSED = SPR_G2_BEGIN + 87,
-    SPR_G2_TOOLBAR_UNMUTE = SPR_G2_BEGIN + 88,
-    SPR_G2_TOOLBAR_UNMUTE_PRESSED = SPR_G2_BEGIN + 89,
+    SPR_G2_TOOLBAR_MUTE = SPR_G2_BEGIN + 83,
+    SPR_G2_TOOLBAR_MUTE_PRESSED = SPR_G2_BEGIN + 86,
+    SPR_G2_TOOLBAR_UNMUTE = SPR_G2_BEGIN + 87,
+    SPR_G2_TOOLBAR_UNMUTE_PRESSED = SPR_G2_BEGIN + 88,
```

Comment:
Either this should start at 85 (if you're just decrementing 1 from everyone after the disabled sprite) or all of the lines should then follow the 83 and be 84, 85, 86 (not skipping some numbers)

Reply:
Good catch, thanks! I wonder what went wrong there, haha. (Also, if the booster pieces for the junior roller coaster are unused, why are they still compiled into `g2.dat`…?)

Reply:
I really don't know :P

Reply:
The booster pieces for the Junior RC are most definitely used. I'm curious as to why it appears they aren't.